RevitLookup/Core/Streams/ElementMethodsStream.cs

```diff
@@ -30,6 +30,7 @@ public void Stream(Type type)
 
         foreach (var methodInfo in methods)
         {
+            if(IsMethodExcept(methodInfo)) continue;
```

Comment:
Open `RevitLookup/Core/CollectorExtensions/DataFactory.cs`

RollBack your changes and add the following line

```c#
if (declaringType == typeof(PrintManager) && info.Name == nameof(PrintManager.SubmitPrint))
    return null;
```

Reply:
i think it ok, I will change it to return value true and change code base.